modules/@apostrophecms/soft-redirect/index.js

```diff
@@ -58,36 +58,26 @@ module.exports = {
           }
         }
       },
-      '@apostrophecms/page:beforeSend': {
```

Comment:
@boutell I see no issue not using this event anymore since we are now systematically adding the url before docs are saved.

But I might be missing a scenario where we'd still need to add the url when visiting a page (ie: before sending a page)